pythonforandroid/recipes/greenlet/__init__.py

```diff
@@ -1,11 +0,0 @@
-from pythonforandroid.recipe import CompiledComponentsPythonRecipe
-
-
-class GreenletRecipe(CompiledComponentsPythonRecipe):
-    version = '0.4.15'
-    url = 'https://pypi.python.org/packages/source/g/greenlet/greenlet-{version}.tar.gz'
-    depends = ['setuptools']
-    call_hostpython_via_targetpython = False
-
-
-recipe = GreenletRecipe()
```

Comment:
I think that this recipe shouldn't be removed, since it compiles a library which will link to the wrong system libraries, for instance, if we inspect the library generated for an app built without this recipe:
```
opacam@DEPOBOX:/media/DEVEL/Android/python-for-android/dists/bdist_unit_tests_app__arm64-v8a/_python_bundle/_python_bundle/site-packages$ readelf -d greenlet.so

La section dynamique à l'offset 0x6020 contient 25 entrées :
  Seny       Tipus                        Nom/Valor
 0x0000000000000001 (NEEDED)             Biblioteca compartida: [libpthread.so.0]
 0x0000000000000001 (NEEDED)             Biblioteca compartida: [libc.so.6]
...
```

As you see, it will depend of system versioned libraries...I don't think that it will work on android...right?

Reply:
Good catch! I've reverted that one.
Funny to see your system is a mix of french and Spanish by the way :smile:

Reply:
Hahaha, my system doesn't translate to Catalan in my terminal and mix english, french and spanish :laughing: